livekit-agents/livekit/agents/job.py

```diff
@@ -79,7 +79,9 @@ def room(self) -> rtc.Room:
     def agent(self) -> rtc.LocalParticipant:
         return self._room.local_participant
 
-    def add_shutdown_callback(self, callback: Callable[[], Coroutine]) -> None:
+    def add_shutdown_callback(
+        self, callback: Callable[[], Coroutine[None, None, None]]
```

Comment:
Wondering what is the right type here.
Awaitable[Any]?

Reply:
Awaitable doesn't work downstream when we do an `asyncio.create_task` which wants a Coroutine